lib/commands/ls.js

```diff
@@ -58,6 +58,7 @@ class LS extends ArboristWorkspaceCmd {
     const unicode = this.npm.config.get('unicode')
     const packageLockOnly = this.npm.config.get('package-lock-only')
     const workspacesEnabled = this.npm.flatOptions.workspacesEnabled
+    const includeWorkspaceRoot = this.npm.flatOptions.includeWorkspaceRoot
```

Comment:
The reason workspacesEnabled is pulling from flatoptions is because it's derived from `workspaces` config.  includeWorkspaceRoot has no such distinction so we should pull it from config like the rest of this command does.

We know this is less than ideal, but fixing this disparity is a future concern.